src/routes/_layout.clients.lazy.tsx

```diff
@@ -1,13 +1,50 @@
-import { createLazyFileRoute } from "@tanstack/react-router";
+import { databases } from "@/appwrite";
+import { type Client, columns } from "@/components/clients/columns";
+import { Button } from "@/components/ui/button";
+import { DataTable } from "@/components/ui/data-table";
+import { useQuery } from "@tanstack/react-query";
+import { Link, createLazyFileRoute } from "@tanstack/react-router";
+import { Search } from "lucide-react";
+
+async function getData(): Promise<Client[]> {
+  const clients = await databases.listDocuments("default", "clients");
+  return clients.documents.map((client) => ({
+    name: client.name,
+    company: client.company,
+    address: client.address1,
+    city: client.city,
+    zipcode: client.zip,
+    state: client.state,
+  }));
```

Comment:
The schema in the code is pretty flexible and we usually update it to match what the API provides so that we don't have to do this annoying mapping.